### PR TITLE
Change bridges to use token inflationMultipliers

### PR DESCRIPTION
**Pull request by albertnbrown**
If the bridge inflationMultiplier is desynced from the token on the same chain, incorrect messages will be sent. With this change, the source of truth is the token and the bridge trusts it completely for the value.

### CODE REVIEW

File: contracts/bridge/L2ECOBridge.sol
Comment (nknavkal) [inline contracts/bridge/L2ECOBridge.sol:1]: i think we keep the inflation multiplier here and update it on calls to rebase to save gas, just one less external call per withdrawal

**Review by nknavkal — COMMENTED** [review]
looks good but see comment

File: contracts/bridge/L2ECOBridge.sol
Comment (albertnbrown) [reply contracts/bridge/L2ECOBridge.sol:1]: Everywhere the variable is used could be a place where it could be de-synced

Comment (nknavkal) [reply contracts/bridge/L2ECOBridge.sol:1]: ah i see where the issue is

**Comment by albertnbrown** [comment]
merged by the deploy branch sync https://github.com/eco-association/op-eco/pull/47

**Pull request closed without merging**